concepts/classes/about.md

```diff
@@ -0,0 +1,260 @@
+# About
+
+JavaScript includes the capabilities for object-oriented programming ([OOP][wiki-oop]).
+In OOP, you want to create objects (_instances_) from "templates" (_classes_) so that they include certain data and functionality.
+The data properties are called _fields_ in the OOP context, the function properties are called _methods_.
+
+JavaScript did not have classes at all before they were added to the language specification in 2015.
+And even though a `class` keyword is available nowadays, JavaScript is still a _prototype-based_ language.
+
+To understand what that means and how JavaScript actually works, we will go back to the time when there were no classes.
```

Comment:
```suggestion
To understand what it means to be a prototype-based language and how JavaScript actually works, we will go back to the time when there were no classes.
```

I recommend we don't refer back to the previous sentences because the _that_ is slightly ambiguous.